public/js/app/components/statistics/domain/schools.js

```diff
@@ -4,9 +4,10 @@ const SCHOOLS = {
   EECS: 'EECS',
   ITM: 'ITM',
   SCI: 'SCI',
+  ALLSCHOOLS: 'All schools',
```

Comment:
`All schools`  better to be replaced with `allSchools`

Reply:
done